src/mapper/babbage.rs

```diff
@@ -142,15 +144,27 @@ impl EventWriter {
                 false => None,
             },
             transactions: None,
+            invalid_transactions: None
         };
 
         if self.config.include_block_details {
             record.transactions = Some(self.collect_babbage_tx_records(source)?);
         }
 
+        if self.config.include_invalid_transaction_details {
+            record.invalid_transactions = self.collect_invalid_tx_indices(source);
+        }
+
```

Comment:
I don't think we need to hide this under a config flag. If the computational / storage cost of a field is relatively cheap, we should include it by default. Mapper config flags make sense in scenarios where we're dealing with heavy data that might have an impact on performance.

In this scenario, since we're just recording the indices of the invalid txs, it doesn't seem like there's any sensible performance penalty.

Reply:
> I don't think we need to hide this under a config flag. If the computational / storage cost of a field is relatively cheap, we should include it by default. Mapper config flags make sense in scenarios where we're dealing with heavy data that might have an impact on performance.
> 
> In this scenario, since we're just recording the indices of the invalid txs, it doesn't seem like there's any sensible performance penalty.

this totally makes sense, I guess I was being super conservative to adding new data / field.